src/components/00-materials/story.js

```diff
@@ -194,8 +264,15 @@ export const IconsAndImages = () => {
         margin: 20px 0;
       }
 
-      .item-name {
+      .materials__asset-name {
         margin-left: 5px;
+        margin-right: 10px;
+        width: 210px;
+      }
+
+      .materials__asset-name > p {
+        text-overflow: ellipsis;
```

Comment:
The only purpose for the story is to show the name of the icons in my opinion :). I would never ellippsis them. So that developers can copy-paste the names.

We have enough space on the right. Maybe just not set a width to this row (at line 270)

Reply:
But what if google decides to do this?

`super-duper-ultra-long-and-weird-svg-name-that-does-take-a-lot-of-space.svg`?

There are some super long names and I think, with autocomplete people could still find the icon when they import. The only other solution I see here is to increase the width of the text.

Reply:
I made a compromise: Increased the max-width of the text significantly.